public/components/authentication/authentication.js

```diff
@@ -2,16 +2,13 @@ import React, { useState } from 'react'
 import loadingAnimation from '../../images/loading_animation.gif'
 import {
   RequestUserLoginAuthentication,
-  RequestUserRegisterAuthentication,
   AuthenticateUserInfo,
   ResultStatus,
 } from '../requestService/authRequestService'
 import './authentication.css'
 
-const loginText = 'Sign In'
-const registerText = 'Sign Up'
-const toggleToLoginText = 'Already Have An Account'
-const toggleToRegisterText = 'Create An Account'
+const registerationURL =
+  'https://ucfbrainbeats.b2clogin.com/ucfbrainbeats.onmicrosoft.com/oauth2/v2.0/authorize?p=B2C_1_signup&client_id=037bbefc-958e-489d-ba61-8c0823284010&nonce=defaultNonce&redirect_uri=https%3A%2F%2Fbrain-beats-server-docker.azurewebsites.net%2F.auth%2Flogin%2Faad%2Fcallback&scope=openid&response_type=id_token&prompt=login'
```

Comment:
Could we not have this hardcoded? The next team is gonna have to redeploy resources and this might change